### ISSUE TITLE
nan loss while lora finetuning

### ISSUE THREAD

**Issue by Kartik-3004** [THUDM/CogVideo#745]
I am getting nan loss value while finetuning CogVideo t2v 5B model. 

`Training steps:   0%|                                                     | 23/92410 [04:32<302:43:29, 11.80s/it, grad_norm=nan, loss=nan, lr=4.6e-8]03/24/2025 14:56:36 - WARNING - tensorboardX.x2num - NaN or Inf found in input tensor.
03/24/2025 14:56:36 - WARNING - tensorboardX.x2num - NaN or Inf found in input tensor.
Loss:  Loss:  Loss:  tensor(nan, device='cuda:0', dtype=torch.float64, grad_fn=<MeanBackward0>)
tensor(nan, device='cuda:1', dtype=torch.float64, grad_fn=<MeanBackward0>)
tensor(nan, device='cuda:2', dtype=torch.float64, grad_fn=<MeanBackward0>)`

I am using https://github.com/THUDM/CogVideo/blob/main/finetune/train_ddp_t2v.sh to finetune with all default parameters. The only thing I changed is the input data. 

Can you help resolve the issue ?

**Comment by Kartik-3004** [comment]
I am using the default —mixed_precision which is “bf16”

**Comment by Kartik-3004** [comment]
I am still not able to resolve the issue of nan loss when lora finetuning t2v model using train_ddp_t2v.sh
